mbed_flasher/flash.py

```diff
@@ -193,20 +193,15 @@ def flash_multiple(self, build, platform_name,
         for item in device_mapping_table:
             self.logger.info(item['target_id'])
 
-        i = 1
         for device in device_mapping_table:
             ret = self.flash(build=build,
                              target_id=device['target_id'],
                              platform_name=None,
                              device_mapping_table=device_mapping_table,
                              method=method,
                              no_reset=no_reset)
-            if ret == EXIT_CODE_SUCCESS:
-                self.logger.debug("dev#%i -> SUCCESS", i)
-            else:
-                self.logger.warning("dev#%i -> FAIL", i)
+            if ret != EXIT_CODE_SUCCESS:
```

Comment:
this is actually breaking change: earlier we tried to flash all boards even some fails. Now we stop immediately when some fails. I'm fine with this change, but this need to be documented at least.

Reply:
I'll change this back.